x-pack/plugins/aiops/public/components/log_categorization/log_categorization_for_flyout.tsx

```diff
@@ -46,13 +50,19 @@ import { useValidateFieldRequest } from './use_validate_category_field';
 import { FieldValidationCallout } from './category_validation_callout';
 import { CreateCategorizationJobButton } from './create_categorization_job';
 
+enum SELECTED_TAB {
```

Comment:
Or

- `Patterns that occur in the dashboard time range` 
- `Patterns that occur in the page time range`

`selected` could be ambiguous as it might imply the time range of the cell selected in the anomaly swim lane.

Reply:
Edit to tooltip text LGTM